app/src/main/res/layout/activity_customer_create_account.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/pure_white"
+    tools:context=".CustomerCreateAccount">
+
+    <TextView
```

Comment:
Long message clips on default virtual device: https://i.snipboard.io/w75X0O.jpg. Not sure how to fix - I can't find the error message in the codebase.